automated install/basic-install.sh

```diff
@@ -98,6 +98,12 @@ if command -v apt-get &> /dev/null; then
     return 0
   }
 
+  source /etc/os-release || true
+    if [[ ${PRETTY_NAME} == "Raspbian GNU/Linux 8 (jessie)" ]]; then
```

Comment:
Wheezy uses `/etc/network/interfaces` and I'm betting that `stretch` will keep `dhcpcd`. The RPF (Raspbian) uses that as an assistant for their GUI configuration of Raspbian, and I don't see them changing Pixel now. 

The other fun fact is that they have released Raspbian Jessie for x86 (Live environment only I believe, I don't think you can use it as a full desktop replacement yet.) Haven't the motivation to boot that thing on any hardware I have here to check it out.